src/main/java/com/devonfw/cicd/testapp/hello/logic/Dto.java

```diff
@@ -0,0 +1,25 @@
+package com.devonfw.cicd.testapp.hello.logic;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+/**
+ * Data transfer object class used to map the entities to the database
+ * @author hkayed
+ *
+ */
+public class Dto {
```

Comment:
I am not sure but I think the Devon guide recommends naming the DTO Objects **UserDto**, we are using this naming in other projects I think this would be nice for consistency.